CHANGELOG.md

```diff
@@ -171,6 +182,14 @@ These should not affect the functionality of the module.
 - Remove docker nodesets [\#257](https://github.com/voxpupuli/puppet-selinux/pull/257) ([bastelfreak](https://github.com/bastelfreak))
 - drop EOL OSs; fix puppet version range [\#256](https://github.com/voxpupuli/puppet-selinux/pull/256) ([bastelfreak](https://github.com/bastelfreak))
 
+## [1.5.2](https://github.com/voxpupuli/puppet-selinux/tree/1.5.2) (2018-07-13)
```

Comment:
We already have a 1.5.2 version. I think the only closed issue here shouldn't be moved from where it was.

Reply:
Ah, not sure what's going on with the script :-/

Reply:
Oh, I had a leftover tag from another project in a different remote :-/. That's fancy.

Reply:
Sorry about that, hopefully this is fixed now. It looks like one of the rake tasks may have decided to sync my local tags